README.md

```diff
@@ -1,3 +1,19 @@
-## Organizing a Collaborator Summit
+# Node.js Collaborator Summit
+
+The Node.js Foundation hosts regular summits for active contributors to the node.js project, this repository is being used to organize them.
+
+## Cost
+Free!
+
+Although the summits may be coordinated with other events/confrences, you do not need to pay/attend those events to participate in a summit.
+
+## Upcoming Events
+- [Aug 17-18 2016, Amsterdam, Netherlands](https://github.com/nodejs/summit/issues/16)
+- Dec 01-02 2016, Austin, Texas, USA
+
+## Past Events
+- [Aug 06-07 2015, San Francisco, California](https://github.com/nodejs/summit/issues/1)
+
+## Code of Conduct
+See https://github.com/nodejs/summit/issues/2
```

Comment:
I'm glad you looked at that one 👍  as noted in #2, I assume that applies to the Collab Summit- but I don't like acting on assumptions.. ESPECIALLY on things like that.
If maybe at least 1 other person can say they agree I'll drop it in...? Otherwise, I have a reminder to merge this no-matter-what on the 9th. (additional PRs can then update it)